homeassistant/components/seventeentrack/sensor.py

```diff
@@ -157,11 +166,12 @@ def _async_create_remove_entities():
         for status, summary_data in coordinator.data.summary.items()
     )
 
-    _async_create_remove_entities()
-
-    config_entry.async_on_unload(
-        coordinator.async_add_listener(_async_create_remove_entities)
-    )
+    if not config_entry.data.get(DEPRECATED_KEY):
```

Comment:
I'd rather check for entities that are used in automations and scripts and create issues as long as those remain.

Example:
https://github.com/home-assistant/core/blob/8bc9fd23bbe9d7ddcc3d190b327cdb4a2dc4521b/homeassistant/components/habitica/sensor.py#L261-L285